chart/develop/templates/cron.yaml

```diff
@@ -0,0 +1,306 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: svc-questionnaire-scheduler-daily-at-one-am
+  namespace: {{ .Values.namespace }}
+spec:
+  schedule: "0 1 * * *"
+  successfulJobsHistoryLimit: 1
+  failedJobsHistoryLimit: 3
+  concurrencyPolicy: Replace
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          restartPolicy: Never
+          imagePullSecrets:
+            - name: {{ .Values.gitlabAuthSecretName }}
+          containers:
+          - name: cron-container
+            image: registry.gitlab.com/researchable/general/docker-images/curl:latest
```

Comment:
This is a private image, are you sure this shouldn't be configurable?

Reply:
Yes I'm sure. That's what we have the gitlabAuth for.

Reply:
Fine, I'll change it.

Reply:
Fixed.